packages/client/utils/getShuffledArr.ts

```diff
@@ -0,0 +1,10 @@
+const getShuffledArr = <T>(arr: T[]): T[] => {
```

Comment:
-1 I didn't want to have this file duplicated, It exists already in `server/utils` and we're fine using `parabol-client/utils` imports in server. So please remove the version in `server/utils`

Reply:
Ah, I didn't notice we had exactly same function, removed